### ISSUE TITLE
"Add/Remove" feature: selection on click

### ISSUE THREAD

**Issue by mikulabc** [devgianlu/KeywordManagerApp#3]
Leave empty to show all
Show only lines containing
Show all lines with word count (1,2..)
Show all lines with char count (1,2..)
Remove all lines containing word count (1,2..)
Remove all lines containing char count (1,2..)
Add text at the end
Add text at the beginning
Add text in between words
Remove lines containing
Remove lines not containing
Remove text at the end
Remove text at the beginning
Remove text from lines
Replace words (current,new)

**Comment by devgianlu** [comment]
These are filters, right? They can be stacked (many at once) and they effect only what we see as they can be removed.

**Comment by mikulabc** [comment]
no, these here work only in combination with the input field as seen in screenshot no. 1 in issue #1
